cijoe/configs/ax42-freebsd.toml

```diff
@@ -0,0 +1,144 @@
+# The SSH options are passed verbatim to paramiko; see
+# https://www.paramiko.org/ This is common CIJOE infrastructure
+#
+# Used by: cijoe.run() / cijoe.get() / cijoe.put()
+[cijoe.transport.qemu_guest]
+hostname = "localhost"
+port = 4200
+username = "root"
+password = ""
+
+[cijoe.run]
+shell = "csh"
```

Comment:
Change to "sh" if root shell is no longer csh on FreeBSD 14? :)

Reply:
Other stuff breaks in that case with the fio build, some assumption on something...

Reply:
ah well .... 😅